### DISCUSSION TITLE
Data Management

### DISCUSSION THREAD

**Discussion by jbcurtin** [dask/dask#7945] [Q&A]
Hi, I'm curious about the current roadmap for data management in Dask. Is there a document outlining how data is distributed and what features are planned for the future of data scheduling/scatter/distribution?

**Comment by mrocklin** [comment]
Hi Joe, can you say a bit more about what you mean by data management and
data scheduling?

These terms can mean a variety of different things to different people.

**Reply by jbcurtin** [reply]
Hi Matt, yeah. That is part of the question I'm trying to iron out. I'm not sure how to ask the question correctly and I am trying to identify the terms I should be using here.

I used the word `scatter` because I'm aware of [Data Scatter](https://distributed.dask.org/en/latest/locality.html#data-scatter), but I'm not aware of the overall architectural design or roadmap to implement data-transmission to different workers. Is there one available?

**Comment by mrocklin** [comment]
Here is one documentation page that might help?:
https://distributed.dask.org/en/latest/memory.html , but in general the
question "how does dask handle data?" has lots of complex answers.  There
isn't a single overarching explanation for everything that it does.

Maybe this page is useful?  If not you might want to look through the other
pages at distributed.dask.org (which tends to be more technical than
docs.dask.org)  If you have a specific problem that you're running into
then I encourage you to share it.  Then folks here will be more effective
at pointing you in the right direction.
